plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -87,6 +84,51 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 				tabPanel.getSize('height') - (padding.bottom + padding.top);
 
 			var iframe = new CKEDITOR.dom.element('iframe');
+
+			var data = this.codeMirrorEditor.getValue();
+
+			iframe.on('load', function () {
```

Comment:
As @julien pointed out we should extract this to a named function. I'm also wondering why we need this code to be executed twice, since we're running the exact same stuff starting at line 144 and again on this `load` event. Is this necessary?

Reply:
no it's not necessary Carlos, I checked the diff quickly and assumed the code had been moved, not duplicated - thanks for pointing this out.

Reply:
hey @carloslancha , it is not necessary. 
It was Friday so I wanted to finish it quickly, so it seems I forgot to remove the code I moved :D.